packages/grid/_modules_/grid/components/GridViewport.tsx

```diff
@@ -15,13 +14,14 @@ import { GridRenderingZone } from './GridRenderingZone';
 import { GridRow } from './GridRow';
 import { GridRowCells } from './GridRowCells';
 import { GridStickyContainer } from './GridStickyContainer';
+import {
+  gridContainerSizesSelector,
+  gridViewportSizesSelector,
+  gridScrollBarSizeSelector,
+} from '../hooks/utils/sizesSelector';
```

Comment:
they should go next to the hook `useGridContainerSizes` and be called `gridContainerSizesSelector`

Reply:
There isn't a hook called `useGridContainerSizes` or at least I'm not seeing it. That's why I added them in the utils because there is also an `optionsSelector` selector in there so it seemed like the correct place to move it.

Reply:
`useGridContainerProps` is in hooks/root